Kiwix.xcodeproj/project.pbxproj

```diff
@@ -540,11 +543,11 @@
 			isa = PBXGroup;
 			children = (
 				97E94B1D271EF250005B0295 /* Assets.xcassets */,
+				9713F7782AE416E5007DD9EC /* CoreKiwix.modulemap */,
```

Comment:
The module map has to be named differently than what's in the install / built dir

See [this](https://developer.apple.com/forums/thread/75674) for additional context